lib/web/apiserver.go

```diff
@@ -505,12 +505,19 @@ func (h *Handler) bindDefaultEndpoints() {
 	// Forwards traces to the configured upstream collector
 	h.POST("/webapi/traces", h.WithAuth(h.traces))
 
-	// Web sessions
-	h.POST("/webapi/sessions/web", httplib.WithCSRFProtection(h.WithLimiterHandlerFunc(h.createWebSession)))
+	// App sessions
 	h.POST("/webapi/sessions/app", h.WithAuth(h.createAppSession))
-	h.DELETE("/webapi/sessions", h.WithAuth(h.deleteSession))
-	h.POST("/webapi/sessions/renew", h.WithAuth(h.renewSession))
 
+	// Deprecated web sessions routes
```

Comment:
Suggestion:

```suggestion
	// DELETE IN 13, deprecated/unused web sessions routes (avatus)
```

We do grep for those (at least I do), so it's a useful mark. Username added for context, although it also makes it easy to find your own cleanups ;)